### DISCUSSION TITLE
DhaqsoPay Baahida oo xalinaa Hoos ka  fiiri

### DISCUSSION THREAD

**Discussion by ducesane** [dugsiiyeinc/dhaqsoPay#51] [General]
![Capturee45454545](https://github.com/user-attachments/assets/5fafa6ea-9357-447a-9c7f-7d3270701567)

**Comment by sharafdin** [comment]
akhii goordhow muu soo dhigay ?

**Reply by ducesane** [reply]
Haaa

**Comment by A-M-Ahmed** [comment]
haa qorlka hadaa fahmy 😂 waa ok hadba dadka meel ay adan la yhin marka koleyto suuq wu yelanaa project gaan
